backend/endpoints/rooms.py

```diff
@@ -238,11 +238,14 @@ async def join_room(
         )
 
     member = room.get("room_members").get(str(member_id))
+    if member == None:
+        raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail="only existing members can add new members",
+            )
 
     if room["room_type"].upper() == RoomType.CHANNEL:
-        if room["is_private"] is True and (
-            member is None or member["role"].lower() != Role.ADMIN
-        ):
+        if room["is_private"] is True and (member["role"].lower() != Role.ADMIN):
```

Comment:
@jayjaytentacion what was the bug and how does this fix it?

Reply:
The bug is If a non existing member_id or one that isn't a member of a room, is used to make a request to add  new member(s), it's adds new member(s)
Line 241 - line 245 is solving it

Reply:
Okay, makes sense. 
Is that all there is to this issue? What happened to the attached sub-tissues?

Reply:
I tested them  and they where working fine

Reply:
@mikengr

Reply:
https://chat.zuri.chat/docs#/rooms/remove_member_api_v1_org__org_id__rooms__room_id__members__member_id__patch

Reply:
i created a channel, then i used  postman to and and remove members

Reply:
@mikengr